src/zope/app/apidoc/static.py

```diff
@@ -256,316 +314,297 @@ def sendMessage(self, msg, verbosity=4):
+
+    ######################################################################
+    # Retrieval
+
+    retrieval = parser.add_argument_group(title="Retrieval",
+                                          description="Options that deal with setting up the generator")
+
+    ret_kind = retrieval.add_mutually_exclusive_group()
+
+    ret_kind.add_argument(
+        '--publisher', '-p', action="store_const", dest='ret_kind',
+        const="publisher", default='publisher',
+        help="""Use the publisher directly to retrieve the data. The program will bring up
+        Zope 3 for you.
+        """
+    )
+
+    ret_kind.add_argument(
+        '--webserver', '-w', action="store_const", dest='ret_kind',
+        const="webserver",
+        help="""Use an external Web server that is connected to Zope 3."""
+    )
+
+    retrieval.add_argument(
+        '--url', '-u', action="store", dest='url',
+        default="http://localhost/",
+        help="""The URL that will be used to retrieve the HTML pages. This option is
+        meaningless, if you are using the publisher as backend. Also, the value of
+        this option should *not* include the `++apidoc++` namespace."""
+    )
+
+    retrieval.add_argument(
+        '--startpage', '-s', action="store", dest='startpage',
+        default='/++apidoc++/static.html',
+        help="""The startpage specifies the path (after the URL) that is used as the starting
+        point to retrieve the contents. This
+        option can be very useful for debugging, since it allows you to select
+        specific pages. """
+    )
+
+    retrieval.add_argument(
+        '--username', '--user', action="store", dest='username',
+        default="mgr",
+        help="""Username to access the Web site."""
+    )
+
+    retrieval.add_argument(
+        '--password', '--pwd', action="store", dest='password',
+        default="mgrpw",
+        help="""Password to access the Web site."""
+    )
+
+    retrieval.add_argument(
+        '--add', '-a', action="append", dest='additional_urls',
+        nargs="*",
+        default=[
+            '/@@/varrow.png',
+            '/@@/harrow.png',
+            '/@@/tree_images/minus.png',
+            '/@@/tree_images/plus.png',
+            '/@@/tree_images/minus_vline.png',
+            '/@@/tree_images/plus_vline.png',
+        ],
+        help="""Add an additional URL to the list of URLs to retrieve. Specifying those is
+        sometimes necessary, if the links are hidden in cryptic JAvascript code."""
+    )
+
+    retrieval.add_argument(
+        '--ignore', '-i', action="append", dest='ignore_modules',
+        nargs="*",
+        default=['twisted', 'zope.app.twisted.ftp.test'],
+        help="""Add modules that should be ignored during retrieval. That allows you to limit
+        the scope of the generated API documentation."""
+    )
+
+    retrieval.add_argument(
+        '--load-all', '-l', action="store_true", dest='import_unknown_modules',
+        default=True,
```

Comment:
I don't see a `--no-load-all` option, so how can we ever turn this off?

Does argparse automatically add corresponding `--no-xxx` options for arguments with `action="store_true"`?